src/repository/songDAO.js

```diff
@@ -1,4 +1,5 @@
-
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
This should not be in here. I have it in index.js before anything gets set up so all routes should already have access to environment variables